### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by manish-GH**
Added Landing page
[PREVIEW](https://jade-marshmallow-908c07.netlify.app/)

Things to look for

- data folder
- Landing folder in screens

### CODE REVIEW

**Review by Sahil25061999 — APPROVED** [review]
Looks good, really loved the ui, just a suggestion you can increase the spacing between the sections, good naming conventions has been used can be merged.

File: src/screens/Landing/Landing.js [hunk 1]
```diff
@@ -1,182 +1,13 @@
...
-									alt="random author"
-									className="image image-round"
-								/>
-							</div>
-							<h3 className="card-heading">Ayn Rand</h3>
-						</div>
-					</div>
-				</div>
-			</div>
+			<Banner />
```
Comment (Shivani-1524) [inline src/screens/Landing/Landing.js:8]: very clean modularity

File: src/screens/Landing/components/Authors.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { AUTHORS } from "../../../data/landingPageData";
+import { AuthorCard } from "./AuthorCard";
+
+export const Authors = () => {
+	return (
+		<>
+			<h1 className="main-header-author text-center">Featured authors</h1>
+			<div className="author-container">
+				<div className="card-set card-author">
+					{AUTHORS.map((authorData) => (
```
Comment (Shivani-1524) [inline src/screens/Landing/components/Authors.js:11]: why use all caps here? is it a convention

File: src/screens/Landing/components/Categories.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import { BESTSELLERS, RECOMMENDED } from "../../../data/landingPageData";
+import { BookList } from "./BookList";
+import { Link } from "react-router-dom";
+
+export const Categories = () => {
+	return (
+		<>
+			<h1 className="main-header text-center">Discover Your Next Book</h1>
+			<div className="main-content">
+				<h2 className="main-content-header text-center">BESTSELLERS</h2>
```
Comment (Shivani-1524) [inline src/screens/Landing/components/Categories.js:11]: having all caps text on your site is bad for accessibility. people find it hard to read.
https://github.com/humanmade/hm-pattern-library/issues/75

**Review by Shivani-1524 — COMMENTED** [review]
code and ui look good overall,. good to merge

File: src/screens/Landing/components/Authors.js (hunk 2 above)
Comment (manish-GH) [reply src/screens/Landing/components/Authors.js:11]: Yes, for constants it's recommended to use all caps

File: src/screens/Landing/components/Categories.js (hunk 3 above)
Comment (manish-GH) [reply src/screens/Landing/components/Categories.js:11]: Hmm...I've never come across this ever...I'll change this

Comment (manish-GH) [reply src/screens/Landing/components/Categories.js:11]: Actually, I'll come back to this later, it kind of looks odd without the caps

**Pull request merged**